medicinski-razgledi.csl

```diff
@@ -0,0 +1,166 @@
+    <id>http://www.zotero.org/styles/medicinski-razgledi</id>
+    <link href="http://www.zotero.org/styles/medicinski-razgledi" rel="self"/>
+    <link href="http://www.zotero.org/styles/revista-da-sociedade-brasileira-de-medicina-tropical" rel="template"/>
+    <link href="https://medrazgl.si/navodila-sodelavcem/" rel="documentation"/>
+    <author>
+      <name>Luka Petravić</name>
+      <email>lpetravic@me.com</email>
+    </author>
+    <category citation-format="numeric"/>
+    <category field="medicine"/>
+    <issn>0025-8121</issn>
+    <eissn>1581-3797</eissn>
+    <updated>2021-12-06T19:54:16+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <macro name="author">
+    <names variable="author" font-weight="normal" delimiter=",">
+      <name delimiter-precedes-et-al="always" delimiter-precedes-last="always" initialize-with="" name-as-sort-order="all" sort-separator="  "/>
+      <label plural="always" text-case="capitalize-first" strip-periods="true"/>
+    </names>
+  </macro>
+  <macro name="editor">
+    <names variable="editor">
+      <name initialize="true" delimiter=", " delimiter-precedes-last="always" name-as-sort-order="all" sort-separator=" "/>
+      <label prefix=", " plural="contextual" suffix="."/>
+    </names>
+  </macro>
+  <macro name="year-date">
+    <choose>
+      <if variable="issued">
+        <date date-parts="year" form="numeric" variable="issued"/>
+      </if>
+      <else>
+        <text term="no date" form="short"/>
+      </else>
+    </choose>
+  </macro>
+  <macro name="journal-location">
+    <choose>
+      <if type="article-journal article-magazine" match="any">
+        <text variable="volume"/>
+        <text variable="issue" prefix="(" suffix=")"/>
+      </if>
+    </choose>
+  </macro>
+  <macro name="publisher">
+    <group delimiter=", ">
+      <text variable="publisher-place" text-case="title"/>
+      <text variable="publisher"/>
+      <text macro="year-date"/>
+    </group>
+  </macro>
+  <macro name="publisher_thesis">
+    <group delimiter=", ">
+      <text variable="publisher-place"/>
+      <text variable="publisher"/>
+    </group>
+  </macro>
+  <macro name="edition_book">
+    <choose>
+      <if is-numeric="edition">
+        <group delimiter=" ">
+          <number variable="edition" form="numeric"/>
+          <text term="edition" form="short"/>
+        </group>
+      </if>
+      <else>
+        <text variable="edition"/>
+      </else>
+    </choose>
+  </macro>
+  <citation collapse="citation-number">
+    <sort>
+      <key variable="citation-number"/>
+    </sort>
+    <layout font-style="italic" delimiter=", " prefix="(" suffix=")">
+      <text variable="citation-number"/>
+    </layout>
+  </citation>
+<bibliography et-al-min="4" et-al-use-first="3" second-field-align="flush">
+    <sort>
+      <key variable="call-number"/>
```

Comment:
Sorry I hadn't seen this: is this a typo? I'd think you want citation-number ?

Reply:
Sorry I am not really versed in this (I made this style because the journal I was submitting to didn't have it), to what specifically is this aimed at? Could you help me? Thanks!

Reply:
It determines how the bibliography gets sorted. I assume it should just be by order of appearance? Currently it starts by sorting by the call number, i.e. how books are cataloged in a library

Reply:
<img width="715" alt="image" src="https://user-images.githubusercontent.com/29688990/147070543-15337508-8459-43a0-8e40-ebd0d3da36f9.png">
Ok, i have deleted the sort part in accordance with the csl spec sheet, I think it should be ok now (if i understand the spec correctly).